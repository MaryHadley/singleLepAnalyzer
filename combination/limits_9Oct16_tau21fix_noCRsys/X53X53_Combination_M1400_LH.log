000 (1297367.000.000) 10/10 19:27:08 Job submitted from host: <192.168.63.249:41533>
...
001 (1297367.000.000) 10/10 19:27:18 Job executing on host: <192.168.63.231:41752>
...
006 (1297367.000.000) 10/10 19:27:26 Image size of job updated: 1
	3  -  MemoryUsage of job (MB)
	2268  -  ResidentSetSize of job (KB)
...
006 (1297367.000.000) 10/10 19:32:26 Image size of job updated: 283420
	163  -  MemoryUsage of job (MB)
	165960  -  ResidentSetSize of job (KB)
...
006 (1297367.000.000) 10/10 20:12:31 Image size of job updated: 284240
	163  -  MemoryUsage of job (MB)
	166712  -  ResidentSetSize of job (KB)
...
005 (1297367.000.000) 10/10 20:16:12 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:48:32, Sys 0 00:00:00  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:48:32, Sys 0 00:00:00  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	1631  -  Run Bytes Sent By Job
	323  -  Run Bytes Received By Job
	1631  -  Total Bytes Sent By Job
	323  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :     1099        1   3824816
	   Memory (MB)          :      163     3072      3072
...
