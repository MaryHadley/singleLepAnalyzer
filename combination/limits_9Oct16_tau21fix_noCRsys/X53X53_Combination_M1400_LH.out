/var/lib/condor/execute/dir_9195
Total number of likelihood evaluations: 55000101
Total number of likelihood evaluations: 5500101
[INFO]  executing script /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_LH.py
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_LH/quant-toys:0-sig-ce954cc16e.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  Running '/home/ssagir/CMSSW_7_3_0/src/theta/bin/theta /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_LH/quant-data-sig-059e7fae22.cfg --nowarn'
[WARN]  cannot extract x-value for signal process id 'sig', using 0
[INFO]  workdir is /home/ssagir/CMSSW_7_3_0/src/singleLepAnalyzer/x53x53_2015/combination/limits_9Oct16_tau21fix_noCRsys/X53X53_Combination_M1400_LH
