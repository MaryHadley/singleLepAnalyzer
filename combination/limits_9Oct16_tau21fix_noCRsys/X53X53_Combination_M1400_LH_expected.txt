# x; y; band 0 low; band 0 high; band 1 low; band 1 high
         0   0.079308   0.039375    0.16304  0.054753    0.11517
